### PR TITLE
Make voting more generic

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Wallet/Services/WalletService.cs [hunk 1]
```diff
@@ -1096,7 +1096,7 @@ public async Task<IActionResult> Vote(VoteRequest request, CancellationToken can
                     {
                         this.logger.LogInformation($"Casting vote {voteTransactions.IndexOf(vote)}/{voteTransactions.Count}");
 
-                        var voteCharacter = request.Vote.Value ? 1 : 0;
+                        var voteCharacter = request.Vote;
```
Comment (zeptin) [inline src/Stratis.Bitcoin.Features.Wallet/Services/WalletService.cs:1099]: Should we have a maximum length? If the vote string is too long, together with the address string, the vote transaction will be regarded as non-standard

Comment (fassadlr) [reply src/Stratis.Bitcoin.Features.Wallet/Services/WalletService.cs:1099]: Good point! I think lets make it 3 characters?

Comment (zeptin) [reply src/Stratis.Bitcoin.Features.Wallet/Services/WalletService.cs:1099]: Sounds good to me 👍

Comment (fassadlr) [reply src/Stratis.Bitcoin.Features.Wallet/Services/WalletService.cs:1099]: Done

**Review by zeptin — APPROVED** [review]

**Pull request merged**